frontend/src/screens/WaitingRoom.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import Typography from "@material-ui/core/Typography";
+import EnterName from "../components/EnterName";
+import WaitingPlayers from "../components/WaitingPlayers";
+import Button from "@material-ui/core/Button";
+import { MyContext } from "../infra/MyContext.js";
+
+const MOCK_playerList = ["Elisha", "Lahav", "Aviad"];
+class WaitingRoom extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isUserEntered: false,
+      playerList: MOCK_playerList
+    };
+  }
+
+  render() {
+    return (
+      <div className="waiting-room">
+        <MyContext.Consumer>
+          {context => {
+            return (
+              <div>
+                <Typography variant="h1" component="h2" gutterBottom>
+                  Waiting Room - RoomId: {context.state.currentRoom}
+                </Typography>
+                {!context.state.isUserEntered ? (
+                  <>
+                    <EnterName />
+                  </>
+                ) : (
+                  <></>
+                )}
+
+                <Button variant="contained" color="primary">
+                  Start the game
```

Comment:
start is sufficient

Reply:
https://github.com/aviadch/Mafia/pull/5/commits/f489d1aafffdaa14006e57949f8ef33fccdc3b97